### PR TITLE
adding directors.yml

### PR DESCRIPTION
**Pull request by cordoval**
|Q            |A  |
|---          |---|
|Bug Fix?     |no |
|New Feature? |no |
|BC Breaks?   |no |
|Deprecations?|no |
|Tests Pass?  |no |
|Fixed Tickets| #644  |
|License      |MIT|

### CODE REVIEW

File: .formatter.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ header: |
     /*
      * This file is part of the Elcodi package.
      *
-     * Copyright (c) 2014 Elcodi.com
```
Comment (mmoreram) [inline .formatter.yml]: This change should be placed in a separated PR.

File: .gush.yml [hunk 2]
```diff
@@ -8,3 +8,4 @@ table-pr:
     tests_pass: ['Tests Pass?', no]
     fixed_tickets: ['Fixed Tickets', '']
     license: ['License', MIT]
+base: master
```
Comment (mmoreram) [inline .gush.yml]: Same with this

**Comment by mmoreram** [comment]
Nice work, @cordoval, but if you add these new files, you should add them into the array on Extension.

**Comment by mmoreram** [comment]
@cordoval Could you please squash all commits in one? Then, I will merge it :D

**Comment by cordoval** [comment]
@mmoreram

**Comment by xphere** [comment]
Why are there some marked as `lazy: true` but not others?

**Comment by mmoreram** [comment]
Because Symfony's Console Command implementation requires all commands to be instances (even when has dependencies) just for listing them all, every dependency related with Doctrine should be lazy. Otherwise, when your database is not created yet, you will not be able to list your commands.

**Comment by cordoval** [comment]
it is squashed @mmoreram

**Comment by mmoreram** [comment]
Yes ;D

**Comment by mmoreram** [comment]
Returning to @xphere... What philosophy we should follow about `lazy`? All lazies? What do you think @cordoval ?

**Comment by cordoval** [comment]
all lazies should be fine

**Comment by mmoreram** [comment]
Great. Do you want to update them all? Do you want me to do that? Feel free.

**Comment by cordoval** [comment]
I can do it, but can i update that in a separate PR? :older_man:

**Comment by mmoreram** [comment]
why?

**Comment by cordoval** [comment]
just asking, ok i will do the change here.

**Comment by mmoreram** [comment]
:) Thanks @cordoval

**Comment by cordoval** [comment]
@mmoreram

**Comment by mmoreram** [comment]
Waiting for travis :)

File: src/Elcodi/Bundle/EntityTranslatorBundle/Resources/config/directors.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+services:
+
+    #
+    # Directors
+    #
+    elcodi.director.entity_translation:
+        class: Elcodi\Component\Core\Services\ObjectDirector
+        lazy: true
+        arguments:
+            - @elcodi.object_manager.currency
```
Comment (mmoreram) [inline src/Elcodi/Bundle/EntityTranslatorBundle/Resources/config/directors.yml]: currency?

**Comment by mmoreram** [comment]
Plase, make sure all tests are passing.

**Comment by cordoval** [comment]
should be passing now and squashed :+1:

**Pull request merged**